Boyadjie/Uncover
# chore(webapp): setup postcss and clean app

## 🤔 Why?

We want to use postcss style lib and we need to clean the app to start with a great whiteboard

## 💻 How?

I've removed everything from the defsult next.js homepage setup

## ✅ How to validate this PR?

- Try to run `pnpm dev` and see if there is nothing left
- Try using postcss in style
